src/ArborX_LinearBVH.hpp

```diff
@@ -134,6 +134,8 @@ BoundingVolumeHierarchy<DeviceType>::BoundingVolumeHierarchy(
           Kokkos::ViewAllocateWithoutInitializing( "internal_and_leaf_nodes" ),
           _size > 0 ? 2 * _size - 1 : 0 )
 {
+    Kokkos::Profiling::pushRegion( "ArborX:BVH:construction" );
```

Comment:
I suppose this excludes the memory allocation for the nodes.  Please add a comment.

Reply:
That's a fair point, as it implicitly assumes 0 associated cost. I think it's fine as it is, as otherwise will need to move that View creation inside the constructor. Will add a comment.